src/Plugin/Config/PhpcqConfigOptionsBuilder.php

```diff
@@ -15,7 +15,7 @@ final class PhpcqConfigOptionsBuilder implements ConfigOptionsBuilderInterface
     public function describeArrayOption(
         string $name,
         string $description,
-        array $defaultValue,
+        array $defaultValue = [],
```

Comment:
This makes it impossible to have no default at all. I think we should make this nullable as well, as an empty array is not always correct.

Reply:
Okay, I changed it. I couldn't imagine a case there it would be needed